sdk/cosmos/azure-cosmos/test/test_auto_scale.py

```diff
@@ -59,7 +59,6 @@ def test_auto_scale(self):
         self.assertEqual(
             created_container_properties.auto_scale_max_throughput, 7000)
         self.assertEqual(created_container_properties.auto_scale_increment_percent, 0)
-        self.assertEqual(created_container_properties.offer_throughput, None)
```

Comment:
Why was this removed?

Reply:
I messaged you about this previously this was removed because it will not work due to change of logic in deserialize throughput

Reply:
This seems kind of a bug... why does the service return both the autoscale values _and_ a default throughput?

Reply:
As I mentioned in chat - let's perhaps restore the deserialization logic so that we don't have to modify this test in order to unblock this PR/beta release. I think we need to open an issue to explore this more fully before GA.